java-squid/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -672,7 +680,19 @@ boolean isInheritedIn(JavaSymbol symbol, JavaSymbol.TypeJavaSymbol clazz) {
       case Flags.PUBLIC:
         return true;
       case Flags.PRIVATE:
-        return symbol.owner() == clazz;
+        // FIXME take only constructors into account when all the members should be JLS-6.6.1
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)